.github/CODEOWNERS

```diff
@@ -2647,6 +2647,116 @@ x-pack/plugins/observability_solution/synthetics/server/saved_objects/synthetics
 /.github/workflows/oblt-github-commands   @elastic/observablt-robots @elastic/kibana-operations
 /.github/workflows/undeploy-my-kibana.yml @elastic/observablt-robots @elastic/kibana-operations
 
+# Sustainable kibana - Temporary placeholder for new folders
+# see https://github.com/elastic/kibana/pull/203682
+
+packages/deeplinks/ml @elastic/ml-ui
+packages/default-nav/ml @elastic/ml-ui
+packages/kbn-esql-ast @elastic/kibana-esql
+packages/kbn-esql-editor @elastic/kibana-esql
+packages/kbn-esql-utils @elastic/kibana-esql
+packages/kbn-esql-validation-autocomplete @elastic/kibana-esql
+packages/kbn-language-documentation @elastic/kibana-esql
+packages/kbn-osquery-io-ts-types @elastic/security-asset-management
```

Comment:
The paths above seem "legacy"

Reply:
Yeah I think you are correct. I'm not sure where it came from when I merged [all your PR](https://github.com/elastic/kibana/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc+Sustainable+Kibana+Architecture%3A+Move+modules) changes into one. But if you can easily generate a list of paths for all inevitable changes that would be better.